### DISCUSSION TITLE
[Question]: Frigate+ clarification request--should a trailer be coded as a car?

### DISCUSSION THREAD

**Discussion by tbclark3** [blakeblackshear/frigate#11473] [Ask A Question] [answered]
### What is your question:

For frigate+, should the trailer in this photo count as a car? In this case, there is a truck in front of it though no part of the truck is visible. Sometimes trailers like this are left on the street for a period of time with no truck attached, so I don't think the trailer would be a car in that case. The same question might apply to a boat trailer without a visible truck or car, in which case the object should probably be considered a boat rather than a car.  I am asking in the interest of consistency of the Frigate+ model more than my personally created model.

![01HY95SXYQHM8T0V5DEQQ8W524](https://github.com/blakeblackshear/frigate/assets/4080951/7b929bb1-10b4-4e2c-9d31-e78f26d2f294)

**Comment by NickM-27** [comment, marked as answer]
personally I never include a trailer as a car regardless, it is only ever the car that is labeled as car
